src/content_scripts/applyColorSchemeJs.js

```diff
@@ -355,6 +367,7 @@ function applyJsOverwrite() {
     });
 
     overwroteMatchMedia = true;
+    console.log(ADDON_FAKED_WARNING);
```

Comment:
As far as I see this logs a message on each site, basically, as the add-on modifies that everywhere.
Sorry, but this is not what I intended.

As you can see in https://github.com/rugk/website-dark-mode-switcher/issues/11, there was quite some discussion about too many logs by this add-on, so I've modified it to only log something _when the add-on actively does sth._, i.e. if “it fakes the result of a JS”.
So better only print that log when the website accesses the “faked” API.